ai-platform-unified/notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb

```diff
@@ -0,0 +1,1560 @@
+{
```

Comment:
<p>Put the names of the fields in code font.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/356/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Done